src/axom/sidre/docs/sphinx/group.rst

```diff
@@ -9,53 +9,53 @@
 
-.. note:: Group and View objects can only be created and destroyed using
-          Group methods provided for this. The Group and View constructors 
-          and destructors are private. 
+.. note:: ``Group`` and ``View`` objects can only be created and destroyed using
+          ``Group`` class methods. The ``Group`` and ``View`` class 
+          constructors and destructors are private. 
 
-A Group or View object is owned by the Group that created it; i.e., its parent 
-Group or *owning* Group, respectively. Groups and Views maintain pointers to 
-their parent/owning Group. Thus, one may *walk* up or down a Group hierarchy
-to access different Groups and Views in it.
+A group or view is owned by the group that created it; i.e., its parent group 
+or *owning* group, respectively. Groups and views maintain pointers to 
+their parent/owning group. Thus, one may *walk* up or down a group hierarchy
+to access groups and views in it, as needed.
 
-.. note:: * The name (string) of a Group or View **must be unique** within its 
+.. note:: * The name (string) of a group or view **must be unique** within its 
             parent/owning Group.
-          * A Group or View has a unique integer identifier within its
+          * A group or view has a unique integer identifier within its
             parent/owning group, which is generated when it is created.
-          * Views and child Groups in a Group can be accessed by name or
+          * Views and child groups in a group can be accessed by name or
             integer id.
 
-A Group object can be moved or copied to another Group. When a Group is moved
-to another Group, it is removed from its original parent and the Group to 
+A group can be moved or copied to another group. When a group is moved
+to another group, it is removed from its original parent and the group to 
 which it is moved becomes its parent. This implies that the entire subtree 
-of Groups and Views within the moved Group is moved as well and can no longer 
-be accessed via the original parent Group. When a Group is copied to another 
-Group, a copy of the entire Group subtree rooted at the copied Group is added
-to the Group to which it is copied. A **shallow** copy is performed for the
-data in each View; i.e., a new View object is created in the destination, but 
-the data is shared by the original and new View.
-
-.. note:: View copy operations perform **shallow** copies of the View data.
-
-Some methods for creating, destroying, querying, and retrieving Groups and
-Views take a string with *path syntax*, where parent and child Group names
-are joined with the path separator character, '/'.
-Other methods take the name of an
-immediate child of a Group. Methods that require the name of a direct child 
-are marked with 'Child' in their name, such as ``hasChildView()`` and 
+of groups and giews within the moved group is moved as well and can no longer 
```

Comment:
I'm sure you meant
```suggestion
of groups and views within the moved group is moved as well and can no longer
```

Reply:
fixed. although 'view' is an overloaded term....